app/assets/javascripts/discourse/lib/url.js.es6

```diff
@@ -97,10 +97,25 @@ const DiscourseURL = Ember.Object.createWithMixins({
     It contains the logic necessary to route within a topic using replaceState to
     keep the history intact.
   **/
+
+  isInternalPathButNotEmber: function(path){
+   /*
+   RSSwhitelist = ["posts.rss", "support.rss", "bug.rss", "feature.rss", "dev.rss", "ux.rss", "installation.rss",
+   "meta.rss", "marketplace.rss", "hosting.rss", "howto.rss", "plugin.rss", "praise.rss", "blog.rss", "faq.rss", "releases.rss"];
+
+   This array contains some of the .rss routes on Discourse, several of them are categories, and the method is
+   extracting the .rss route and verifying if the indexOf is >= to zero.
+   */
+
+   let pathStripped =  path.split("/").pop();
+   return path.indexOf(pathStripped) >= 0;
```

Comment:
I don't understand how this works. You're taking the last segment from the path, but then checking that it's in the path? Wouldn't it always be true unless it's a root path?

Can you provide more examples?

I would be more comfortable with what @SamSaffron suggested and just check for an extension of `.rss` or `.json`